app/controllers/course/assessment/question/text_responses_controller.rb

```diff
@@ -52,13 +65,30 @@ def destroy
   def text_response_question_params
     params.require(:question_text_response).permit(
       :title, :description, :staff_only_comments, :maximum_grade, :allow_attachment,
-      :hide_text,
+      :hide_text, :is_comprehension,
```

Comment:
Can I suggest that we check `is_comprehension` then whitelist the params, so something like ... 
```ruby
def text_response_question_params
  if params[:question_text_response][:is_comprehension]
    <Your added code>
  else
    <Original code>
  end
end
```